admin/sql/create_tables.sql

```diff
@@ -18,4 +18,22 @@ CREATE TABLE listen (
   raw_data        JSONB
 );
 
+CREATE TABLE token (
+     id               SERIAL,
+     user_id          INTEGER, -- FK to "user".id
+     token            TEXT NOT NULL,
+     api_key          VARCHAR NOT NULL,
+     ts               TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+ALTER TABLE token ADD CONSTRAINT token_api_key_uniq UNIQUE (api_key);
```

Comment:
Our policy is to not mix create table and alter statements in this file. If you are doing lookups on this value, perhaps you should have a `UNIQUE INDEX` instead, in which case these statements should be in `create_indexes.sql`